internal/changelog/builder.go

```diff
@@ -161,6 +168,39 @@ func collectFragment(fs afero.Fs, path string, info os.FileInfo, err error, file
 	return nil
 }
 
+func ExtactEventNumber(linkType, eventURL string) (string, error) {
+	urlParts := strings.Split(eventURL, "/")
+
+	if len(urlParts) < 1 {
+		return "", fmt.Errorf("cant get event number")
+	}
+
+	switch linkType {
+	// maybe use regex to validate instead of a simple string check
+	case "pr":
+		if !strings.Contains(eventURL, "pull") {
+			return "", fmt.Errorf("link is invalid for pr")
+		}
+	case "issue":
+		if !strings.Contains(eventURL, "issues") {
+			return "", fmt.Errorf("link is invalid for issue")
+		}
+	}
+
+	return urlParts[len(urlParts)-1], nil
+}
+
+func CreateEventLink(linkType, owner, repo, eventID string) string {
+	switch linkType {
+	case "issue":
+		return fmt.Sprintf("https://github.com/%s/%s/issues/%s", owner, repo, eventID)
+	case "pr":
+		return fmt.Sprintf("https://github.com/%s/%s/pull/%s", owner, repo, eventID)
+	default:
+		return ""
```

Comment:
Instead of returning an empty string here, let's panic. This is a developer error (passing the wrong `linkType` value), returning an empty string would shadow the error, making it hard to back track. By panicking we signals developers the error.

```suggestion
		panic("wrong linkType")
```